addons/sale_management/static/src/interactions/sale_management.js

```diff
@@ -0,0 +1,56 @@
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+
+import { rpc } from "@web/core/network/rpc";
+
+class SaleUpdateLineButton extends Interaction {
+    static selector = ".o_portal_sale_sidebar";
+    dynamicContent = {
+        "a.js_update_line_json": {
+            "t-on-click.prevent": (ev) => this.onClickOptionQuantityButton(ev.currentTarget),
+        },
+        "a.js_add_optional_products": {
```

Comment:
Somehow those class names only seem to appear in tours in enterprise... `TestSaleExternalTaxesSalePortal`
Does this tour even work ? And if it is the case, where does that class come from in there ?

Reply:
Oops I had a filter activated on XML files, forget this comment